src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 400px;
```

Comment:
font-weight: 400px; is invalid, should be without "px".
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/2eb0cf64-3195-434c-8581-257ee45c5b97)